src/main/webapp/paginationScript.test.js

```diff
@@ -0,0 +1,39 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const calculateListAugment = require('./paginationScript');
+
+const TOTAL_CARDS_TO_DISPLAY = 3;
+const MAX_LIST_VIEW_NUMER = 15;
+
+test('It calculateListAugment performs as expected when at beginning of list', () => {
+  currentFirstCardIndex = 0;
```

Comment:
Do you need this variable? I don't see it being used anywhere. Perhaps you intended for calculateListAugment to receive it as a parameter?

Reply:
It's being used in the second test, line 26

Reply:
...Mispelled oops